mods/cnc/maps/gdi06/gdi06.lua

```diff
@@ -0,0 +1,239 @@
+	waypoint7.Location,
+	waypoint6.Location
+}
+
+FootPatrol2Squad = { MiniGunner3, MiniGunner4 }
+FootPatrol2Route = {
+	waypoint14.Location,
+	waypoint16.Location
+}
+
+FootPatrol3Squad = { MiniGunner5, MiniGunner6 }
+FootPatrol3Route = {
+	waypoint15.Location,
+	waypoint17.Location
+}
+
+FootPatrol4Route = {
+	waypoint4.Location,
+	waypoint5.Location
+}
+
+FootPatrol5Squad = { RocketSoldier2, RocketSoldier3, RocketSoldier4 }
+FootPatrol5Route = {
+	waypoint4.Location,
+	waypoint12.Location,
+	waypoint13.Location,
+	waypoint8.Location,
+	waypoint9.Location,
+}
+
+Buggy1Route = {
+	waypoint6.Location,
+	waypoint7.Location,
+	waypoint2.Location,
+	waypoint8.Location,
+	waypoint9.Location,
+	waypoint8.Location,
+	waypoint2.Location,
+	waypoint7.Location
+}
+
+Buggy2Route = {
+	waypoint6.Location,
+	waypoint10.Location,
+	waypoint11.Location,
+	waypoint10.Location
+}
+
+HuntTriggerActivator = { SAM03, SAM04, SAM05, SAM06, LightTank1, LightTank2, LightTank3, Buggy1, Buggy2, Turret1, Turret2 }
+
+AttackCellTriggerActivator = { CPos.New(57,26), CPos.New(56,26), CPos.New(57,25), CPos.New(56,25), CPos.New(57,24), CPos.New(56,24), CPos.New(57,23), CPos.New(56,23), CPos.New(57,22), CPos.New(56,22), CPos.New(57,21), CPos.New(56,21) }
+AttackUnits = { LightTank2, LightTank3 }
+
+KillCounter = 0
+
+WorldLoaded = function()
+	player = Player.GetPlayer("GDI")
+	enemy = Player.GetPlayer("Nod")
+	civilian = Player.GetPlayer("Neutral")
+
+	Trigger.OnObjectiveAdded(player, function(p, id)
+		Media.DisplayMessage(p.GetObjectiveDescription(id), "New " .. string.lower(p.GetObjectiveType(id)) .. " objective")
+	end)
+	Trigger.OnObjectiveCompleted(player, function(p, id)
+		Media.DisplayMessage(p.GetObjectiveDescription(id), "Objective completed")
+	end)
+	Trigger.OnObjectiveFailed(player, function(p, id)
+		Media.DisplayMessage(p.GetObjectiveDescription(id), "Objective failed")
+	end)
+
+	Trigger.OnPlayerWon(player, function()
+		Media.PlaySpeechNotification(player, "Win")
+	end)
+
+	Trigger.OnPlayerLost(player, function()
+		Media.PlaySpeechNotification(player, "Lose")
+	end)
+
+	if Map.Difficulty == "Easy" then
+		CommandoType = "rmbo.easy"
+		KillCounterHuntThreshold = 30
+	elseif Map.Difficulty == "Hard" then
+		CommandoType = "rmbo.hard"
+		KillCounterHuntThreshold = 15
+	else
+		CommandoType = "rmbo"
+		KillCounterHuntThreshold = 20
+	end
+
+	Media.PlayMusic("rain")
+
+	destroyObjective = player.AddPrimaryObjective("Destroy the Nod ********.")
+
+	Trigger.OnKilled(Airfield, function()
+		player.MarkCompletedObjective(destroyObjective)
+	end)
+
+	Utils.Do(NodBase, function(structure)
+		Trigger.OnKilled(structure, function()
+			player.MarkCompletedObjective(destroyObjective)
+		end)
+	end)
+
+	Trigger.OnAllKilled(IslandSamSites, function()
+		TransportFlare = Actor.Create('flare', true, { Owner = player, Location = Flare.Location })
+		Reinforcements.ReinforceWithTransport(player, 'tran', nil, { lstStart.Location, TransportRally.Location }, nil, nil, nil)
```

Comment:
You don't need to set the last three parameters to `nil`, they already default to `nil`.